packages/trader/build/config.js

```diff
@@ -24,19 +24,39 @@ const htmlOutputConfig = () => ({
           },
 });
 
-const htmlInjectConfig = () => ({
-    links: [
-        {
-            path: 'public/images/favicons',
-            glob: '*',
-            globPath: path.resolve(__dirname, '../src/public/images/favicons'),
-            attributes: {
-                rel: 'icon',
-            },
-        },
-    ],
-    append: false,
-});
+// const htmlInjectConfig = () => ({ // commented out in trader/build/constants.js
+//     links: [
+//         {
+//             path: 'favicon.ico',
+//             attributes: {
+//                 rel: 'icon',
+//             },
+//         },
+//         ...[
+//             { name: 'favicon', rel: 'icon', size: '16' },
+//             { name: 'favicon', rel: 'icon', size: '32' },
+//             { name: 'favicon', rel: 'icon', size: '96' },
+//             { name: 'favicon', rel: 'icon', size: '160' },
+//             { name: 'favicon', rel: 'icon', size: '192' },
+//             { name: 'apple-touch-icon', size: '57' },
+//             { name: 'apple-touch-icon', size: '60' },
+//             { name: 'apple-touch-icon', size: '72' },
+//             { name: 'apple-touch-icon', size: '76' },
+//             { name: 'apple-touch-icon', size: '114' },
+//             { name: 'apple-touch-icon', size: '120' },
+//             { name: 'apple-touch-icon', size: '144' },
+//             { name: 'apple-touch-icon', size: '152' },
+//             { name: 'apple-touch-icon', size: '180' },
+//         ].map(({ name, rel, size }) => ({
+//             path: `public/images/favicons/${name}-${size}.png`,
+//             attributes: {
+//                 rel: rel || name,
+//                 sizes: `${size}x${size}`,
+//             },
+//         })),
+//     ],
+//     append: false,
+// });
```

Comment:
Changes to this file are made just for consistency. Neither generateSWConfig, nor htmlInjectConfig from this file are used anywhere in trader

Reply:
If not used, let's remove this, whole thing htmlInjectConfig()